src/Microsoft.AspNetCore.Rewrite/Internal/UrlRewrite/UrlRewriteFileParser.cs

```diff
@@ -14,12 +15,12 @@ namespace Microsoft.AspNetCore.Rewrite.Internal.UrlRewrite
 {
     public static class UrlRewriteFileParser
     {
-        private static readonly TimeSpan RegexTimeout = TimeSpan.FromMilliseconds(1);
+        private static readonly TimeSpan _regexTimeout = TimeSpan.FromMilliseconds(1);
```

Comment:
Revert, this is the correct style for statics/constants. _foo is for member variables.